"use strict";(self.webpackChunknfdi_sections=self.webpackChunknfdi_sections||[]).push([[2889],{771:(t,e,o)=>{o.r(e),o.d(e,{assets:()=>l,contentTitle:()=>r,default:()=>c,frontMatter:()=>d,metadata:()=>s,toc:()=>a});const s=JSON.parse('{"id":"meta/wg_onto/used-ontologies/nfdi4earth","title":"NFDI4Earth","description":"Source NFDI4Earth)","source":"@site/docs/meta/wg_onto/used-ontologies/nfdi4earth.md","sourceDirName":"meta/wg_onto/used-ontologies","slug":"/meta/wg_onto/used-ontologies/nfdi4earth","permalink":"/nfdi-sections/docs/meta/wg_onto/used-ontologies/nfdi4earth","draft":false,"unlisted":false,"editUrl":"https://github.com/okoepler/nfdi-sections/edit/main/docs/meta/wg_onto/used-ontologies/nfdi4earth.md","tags":[],"version":"current","frontMatter":{"title":"NFDI4Earth"},"sidebar":"tutorialSidebar","previous":{"title":"NFDI4DataScience","permalink":"/nfdi-sections/docs/meta/wg_onto/used-ontologies/nfdi4datascience"},"next":{"title":"NFDI4Energy","permalink":"/nfdi-sections/docs/meta/wg_onto/used-ontologies/nfdi4energy"}}');var n=o(4848),i=o(8453);const d={title:"NFDI4Earth"},r=void 0,l={},a=[];function h(t){const e={a:"a",code:"code",p:"p",table:"table",tbody:"tbody",td:"td",th:"th",thead:"thead",tr:"tr",...(0,i.R)(),...t.components};return(0,n.jsxs)(n.Fragment,{children:[(0,n.jsxs)(e.p,{children:["Source: ",(0,n.jsx)(e.code,{children:"/files/meta/Used_Ontologies_at_NFDI.xlsx"})," (sheet: NFDI4Earth)"]}),"\n",(0,n.jsxs)(e.table,{children:[(0,n.jsx)(e.thead,{children:(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.th,{children:"Ontology ID"}),(0,n.jsx)(e.th,{children:"Ontology"}),(0,n.jsx)(e.th,{children:"Domain"}),(0,n.jsx)(e.th,{children:"License"}),(0,n.jsx)(e.th,{children:"Format uncontrolled"}),(0,n.jsx)(e.th,{children:"Format"}),(0,n.jsx)(e.th,{children:"developed / reused / only indexed / only evaluated"}),(0,n.jsx)(e.th,{children:"Modularity"}),(0,n.jsx)(e.th,{children:"Obofoundry"}),(0,n.jsx)(e.th,{children:"OLS URL"}),(0,n.jsx)(e.th,{children:"Bioportal"}),(0,n.jsx)(e.th,{children:"Github"}),(0,n.jsx)(e.th,{children:"File URL"}),(0,n.jsx)(e.th,{children:"Other URL"}),(0,n.jsx)(e.th,{children:"Description"}),(0,n.jsx)(e.th,{}),(0,n.jsx)(e.th,{children:"Useful Links:"}),(0,n.jsx)(e.th,{children:"OBO Foundry"}),(0,n.jsx)(e.th,{children:"NCBO BioPortal"}),(0,n.jsx)(e.th,{children:"OLS ontology search"}),(0,n.jsx)(e.th,{children:"GFBio Terminology Service"})]})}),(0,n.jsxs)(e.tbody,{children:[(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"BCO"}),(0,n.jsx)(e.td,{children:"Biological Collections Ontology"}),(0,n.jsx)(e.td,{children:"biology"}),(0,n.jsx)(e.td,{children:"CC0 1.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"BFO"}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://ontobee.org/ontology/BCO",children:"https://ontobee.org/ontology/BCO"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.ebi.ac.uk/ols/ontologies/bco",children:"https://www.ebi.ac.uk/ols/ontologies/bco"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/BCO",children:"https://bioportal.bioontology.org/ontologies/BCO"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/BiodiversityOntologies/bco",children:"https://github.com/BiodiversityOntologies/bco"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/BCO",children:"https://bioportal.bioontology.org/ontologies/BCO"})}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:'OBO: The Biological Collections Ontology (BCO) is a being developed as an application ontology as part of the Biocode Commons project, within the OBO Foundry framework. The goal of the BCO is to support the interoperability of biodiversity data, including data on museum collections, environmental/metagenomic samples, and ecological surveys.<br/><br/>GFBio: The biological collection ontology includes consideration of the distinctions between individuals, organisms, voucher specimens, lots, and samples the relations between these entities, and processes governing the creation and use of "samples". Within scope as well are properties including collector, location, time, storage environment, containers, institution, and collection identifiers.'}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"Comments:"}),(0,n.jsx)(e.td,{children:"Please (also) check out the DataPlant sheet, there is some overlap ... Gruss, Steffen"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"ChEBI"}),(0,n.jsx)(e.td,{children:"Chemical Entities of Biological Interest Ontology"}),(0,n.jsx)(e.td,{children:"chemistry"}),(0,n.jsx)(e.td,{children:"CC BY 4.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"OBO"}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://obofoundry.org/ontology/chebi.html",children:"https://obofoundry.org/ontology/chebi.html"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.ebi.ac.uk/ols/ontologies/chebi",children:"https://www.ebi.ac.uk/ols/ontologies/chebi"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/CHEBI",children:"https://bioportal.bioontology.org/ontologies/CHEBI"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/ebi-chebi/ChEBI",children:"https://github.com/ebi-chebi/ChEBI"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/CHEBI",children:"https://bioportal.bioontology.org/ontologies/CHEBI"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"http://www.ebi.ac.uk/chebi",children:"http://www.ebi.ac.uk/chebi"})}),(0,n.jsx)(e.td,{children:"GitHub: Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on \u2018small\u2019 chemical compounds. The term \u2018molecular entity\u2019 refers to any constitutionally or isotopically distinct atom, molecule, ion, ion pair, radical, radical ion, complex, conformer, etc., identifiable as a separately distinguishable entity.<br/><br/>Other source: A structured classification of chemical compounds of biological relevance."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"ENVO"}),(0,n.jsx)(e.td,{children:"Environment Ontology"}),(0,n.jsx)(e.td,{children:"environmental"}),(0,n.jsx)(e.td,{children:"CC0 1.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"BFO"}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://obofoundry.org/ontology/envo.html",children:"https://obofoundry.org/ontology/envo.html"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.ebi.ac.uk/ols/ontologies/envo",children:"https://www.ebi.ac.uk/ols/ontologies/envo"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/ENVO",children:"https://bioportal.bioontology.org/ontologies/ENVO"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/EnvironmentOntology/envo",children:"https://github.com/EnvironmentOntology/envo"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://raw.githubusercontent.com/EnvironmentOntology/envo/master/envo.owl",children:"https://raw.githubusercontent.com/EnvironmentOntology/envo/master/envo.owl"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"http://environmentontology.org/",children:"http://environmentontology.org/"})}),(0,n.jsxs)(e.td,{children:["BioPortal: The most up-to-date information about ENVO is available here: ",(0,n.jsx)(e.a,{href:"http://www.obofoundry.org/ontology/envo.html",children:"http://www.obofoundry.org/ontology/envo.html"})," EnvO is an OBO Foundry and Library ontology for the concise, controlled description of environmental entities such as ecosystems, environmental processes, and environmental qualities. It closely interoperates with a broad collection of other OBO ontologies and is used in a diverse range of projects."]}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"FLOPO"}),(0,n.jsx)(e.td,{children:"Flora Phenotype Ontology"}),(0,n.jsx)(e.td,{children:"anatomical"}),(0,n.jsx)(e.td,{children:"CC BY 3.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"OBO"}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://obofoundry.org/ontology/flopo.html",children:"https://obofoundry.org/ontology/flopo.html"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.ebi.ac.uk/ols/ontologies/flopo",children:"https://www.ebi.ac.uk/ols/ontologies/flopo"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/FLOPO",children:"https://bioportal.bioontology.org/ontologies/FLOPO"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/flora-phenotype-ontology/flopoontology",children:"https://github.com/flora-phenotype-ontology/flopoontology"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/flora-phenotype-ontology/flopoontology/tree/master/ontology",children:"https://github.com/flora-phenotype-ontology/flopoontology/tree/master/ontology"})}),(0,n.jsx)(e.td,{}),(0,n.jsxs)(e.td,{children:["BioPortal: The Flora Phenotype Ontology is an ontology of phenotypes reported in Floras. The original version was developed at the pro-iBiosphere Hackathon in Leiden. This is the pre-classified version of the ontology; the original OWL file is at ",(0,n.jsx)(e.a,{href:"https://github.com/flora-phenotype-ontology/flopoontology/blob/master/ontology/flopo.owl",children:"https://github.com/flora-phenotype-ontology/flopoontology/blob/master/ontology/flopo.owl"})," The Flora Phenotype Ontology is generated from the Flora Malesiana, Flora Gabon, Flora of Central Africa, and a collection of Kew's African Floras. Every class in the ontology has at least one taxon annotation. The (draft) taxon annotation are available at ",(0,n.jsx)(e.a,{href:"http://jagannath.pdn.cam.ac.uk/plant/flora/clean-rerun/",children:"http://jagannath.pdn.cam.ac.uk/plant/flora/clean-rerun/"})]}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"NCBITAXON"}),(0,n.jsx)(e.td,{children:"National Center for Biotechnology Information Organismal Classification"}),(0,n.jsx)(e.td,{children:"biology"}),(0,n.jsx)(e.td,{children:"CC0 1.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"OBO"}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://obofoundry.org/ontology/ncbitaxon.html",children:"https://obofoundry.org/ontology/ncbitaxon.html"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.ebi.ac.uk/ols/ontologies/ncbitaxon",children:"https://www.ebi.ac.uk/ols/ontologies/ncbitaxon"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/NCBITAXON",children:"https://bioportal.bioontology.org/ontologies/NCBITAXON"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/obophenotype/ncbitaxon",children:"https://github.com/obophenotype/ncbitaxon"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/NCBITAXON",children:"https://bioportal.bioontology.org/ontologies/NCBITAXON"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.ncbi.nlm.nih.gov/taxonomy",children:"https://www.ncbi.nlm.nih.gov/taxonomy"})}),(0,n.jsx)(e.td,{children:"OBO: The NCBITaxon ontology is an automatic translation of the NCBI taxonomy database into obo/owl.<br/><br/>NCBI: The Taxonomy Database is a curated classification and nomenclature for all of the organisms in the public sequence databases. This currently represents about 10% of the described species of life on the planet."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"OBOE"}),(0,n.jsx)(e.td,{children:"The Extensible Observation Ontology"}),(0,n.jsx)(e.td,{children:"environmental"}),(0,n.jsx)(e.td,{children:"CC BY 3.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/OBOE",children:"https://bioportal.bioontology.org/ontologies/OBOE"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/NCEAS/oboe",children:"https://github.com/NCEAS/oboe"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/OBOE",children:"https://bioportal.bioontology.org/ontologies/OBOE"})}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"BioPortal: The Extensible Observation Ontology (OBOE) is a formal ontology for capturing the semantics of scientific observation and measurement. The ontology supports researchers to add detailed semantic annotations to scientific data, thereby clarifying the inherent meaning of scientific observations"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"PATO"}),(0,n.jsx)(e.td,{children:"The Phenotype And Trait Ontology"}),(0,n.jsx)(e.td,{children:"anatomical"}),(0,n.jsx)(e.td,{children:"CC BY 3.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"BFO"}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://obofoundry.org/ontology/pato.html",children:"https://obofoundry.org/ontology/pato.html"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.ebi.ac.uk/ols/ontologies/pato",children:"https://www.ebi.ac.uk/ols/ontologies/pato"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/PATO",children:"https://bioportal.bioontology.org/ontologies/PATO"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/pato-ontology/pato/",children:"https://github.com/pato-ontology/pato/"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/PATO",children:"https://bioportal.bioontology.org/ontologies/PATO"})}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"Obo & BioPortal: Phenotypic qualities (properties). This ontology can be used in conjunction with other ontologies such as GO or anatomical ontologies to refer to phenotypes. Examples of qualities are red, ectopic, high temperature, fused, small, edematous and arrested."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"PO"}),(0,n.jsx)(e.td,{children:"Plant Ontology"}),(0,n.jsx)(e.td,{children:"biology"}),(0,n.jsx)(e.td,{children:"CC BY 4.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://obofoundry.org/ontology/po.html",children:"https://obofoundry.org/ontology/po.html"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.ebi.ac.uk/ols/ontologies/po",children:"https://www.ebi.ac.uk/ols/ontologies/po"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/PO",children:"https://bioportal.bioontology.org/ontologies/PO"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/Planteome/plant-ontology",children:"https://github.com/Planteome/plant-ontology"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/PO",children:"https://bioportal.bioontology.org/ontologies/PO"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.plantontology.org",children:"https://www.plantontology.org"})}),(0,n.jsx)(e.td,{children:"PO Consortium: The Plant Ontology is a controlled vocabulary (ontology) that describes plant anatomy and morphology and stages of development for all plants. The goal of the PO is to establish a semantic framework for meaningful cross-species queries across gene expression and phenotype data sets from plant genomics and genetics experiments.<br/><br/>BioPortal: The Plant Ontology is a structured vocabulary and database resource that links plant anatomy, morphology and growth and development to plant genomics data. The PO is under active development to expand to encompass terms and annotations from all plants."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"PTO"}),(0,n.jsx)(e.td,{children:"Plant Trait Ontology"}),(0,n.jsx)(e.td,{children:"anatomical"}),(0,n.jsx)(e.td,{children:"CC BY 4.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"BFO"}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://obofoundry.org/ontology/to.html",children:"https://obofoundry.org/ontology/to.html"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.ebi.ac.uk/ols/ontologies/to",children:"https://www.ebi.ac.uk/ols/ontologies/to"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/PTO",children:"https://bioportal.bioontology.org/ontologies/PTO"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/Planteome/plant-trait-ontology",children:"https://github.com/Planteome/plant-trait-ontology"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/PTO",children:"https://bioportal.bioontology.org/ontologies/PTO"})}),(0,n.jsxs)(e.td,{children:["PO, TO, PECO, PSO: ",(0,n.jsx)(e.a,{href:"https://browser.planteome.org/amigo",children:"https://browser.planteome.org/amigo"})]}),(0,n.jsx)(e.td,{children:"BioPortal: A controlled vocabulary to describe phenotypic traits in plants. Each trait is a distinguishable feature, characteristic, quality or phenotypic feature of a developing or mature plant, or a plant part."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"SWEET"}),(0,n.jsx)(e.td,{children:"Semantic Web for Earth and Environment Technology Ontology"}),(0,n.jsx)(e.td,{children:"geo-sciences"}),(0,n.jsx)(e.td,{children:"CC0 1.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/SWEET",children:"https://bioportal.bioontology.org/ontologies/SWEET"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/ESIPFed/sweet",children:"https://github.com/ESIPFed/sweet"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/SWEET&lt;br/&gt;&lt;br/&gt;http://sweetontology.net/sweetAll",children:"https://bioportal.bioontology.org/ontologies/SWEET&lt;br/&gt;&lt;br/&gt;http://sweetontology.net/sweetAll"})}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"BioPortal: The Semantic Web for Earth and Environmental Terminology is a mature foundational ontology that contains over 6000 concepts organized in 200 ontologies represented in OWL. Top level concepts include Representation (math, space, science, time, data), Realm (Ocean, Land Surface, Terrestrial Hydroshere, Atmosphere, etc.), Phenomena (macro-scale ecological and physical), Processes (micro-scale physical, biological, chemical, and mathematical), Human Activities (Decision, Commerce, Jurisdiction, Environmental, Research). Originally developed by NASA Jet Propulsion Labs under Rob Raskin, SWEET is now officially under the governance of the ESIP foundation."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"GEONAMES"}),(0,n.jsx)(e.td,{children:"The Geonames geographical database"}),(0,n.jsx)(e.td,{children:"geo-sciences"}),(0,n.jsx)(e.td,{children:"CC BY 4.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.geonames.org/ontology/ontology_v3.3.rdf",children:"https://www.geonames.org/ontology/ontology_v3.3.rdf"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"http://www.geonames.org/",children:"http://www.geonames.org/"})}),(0,n.jsx)(e.td,{children:"GeoNames: The GeoNames geographical database is available for download free of charge under a creative commons attribution license. It contains over 25 million geographical names and consists of over 11 million unique features whereof 4.8 million populated places and 13 million alternate names. All features are categorized into one out of nine feature classes and further subcategorized into one out of 645 feature codes... GeoNames is integrating geographical data such as names of places in various languages, elevation, population and others from various sources.<br/><br/>W3C: The GeoNames Ontology makes it possible to add geospatial semantic information to the Word Wide Web. All over 6.2 million geonames toponyms now have a unique URL with a corresponding RDF web service. Other services describe the relation between toponyms.<br/><br/>European Comission: The Geonames ontology provides elements of description for geographical features defined in the geonames.org data base"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"GAZ"}),(0,n.jsx)(e.td,{children:"Gazetteer. This ontology is inactive."}),(0,n.jsx)(e.td,{children:"environment"}),(0,n.jsx)(e.td,{children:"CC0 1.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://obofoundry.org/ontology/gaz.html",children:"https://obofoundry.org/ontology/gaz.html"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.ebi.ac.uk/ols/ontologies/gaz",children:"https://www.ebi.ac.uk/ols/ontologies/gaz"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/GAZ?p=summary",children:"https://bioportal.bioontology.org/ontologies/GAZ?p=summary"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/EnvironmentOntology/gaz",children:"https://github.com/EnvironmentOntology/gaz"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"http://environmentontology.github.io/gaz/",children:"http://environmentontology.github.io/gaz/"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.ebi.ac.uk/ols/ontologies/gaz&lt;br/&gt;&lt;br/&gt;http://environmentontology.github.io/gaz/",children:"https://www.ebi.ac.uk/ols/ontologies/gaz&lt;br/&gt;&lt;br/&gt;http://environmentontology.github.io/gaz/"})}),(0,n.jsx)(e.td,{children:"GAZ: GAZ represents places through their names using an ontological approach to promote semantic coherence. All entities in GAZ can be treated as instances of environments (see the ENVO repository)."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"DCAT"}),(0,n.jsx)(e.td,{children:"Data Catalog Vocabulary"}),(0,n.jsx)(e.td,{children:"vocabulary"}),(0,n.jsx)(e.td,{children:"W3C License"}),(0,n.jsx)(e.td,{children:"RDF"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"W3C"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/DCAT",children:"https://bioportal.bioontology.org/ontologies/DCAT"})}),(0,n.jsx)(e.td,{}),(0,n.jsxs)(e.td,{children:["DCAT V2: ",(0,n.jsx)(e.a,{href:"https://www.w3.org/TR/vocab-dcat/&lt;br/&gt;&lt;br/&gt;DCAT",children:"https://www.w3.org/TR/vocab-dcat/&lt;br/&gt;&lt;br/&gt;DCAT"})," V3: ",(0,n.jsx)(e.a,{href:"https://www.w3.org/TR/vocab-dcat-3/",children:"https://www.w3.org/TR/vocab-dcat-3/"})]}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"W3C: DCAT 3 supersedes DCAT 2 [VOCAB-DCAT-2], but it does not make it obsolete. DCAT 3 maintains the DCAT namespace as its terms preserve backward compatibility with DCAT 2. DCAT 3 relaxes constraints and adds new classes and properties, but these changes do not break the definition of previous terms.<br/><br/>Any new implementation is expected to adopt DCAT 3, while the existing implementations do not need to upgrade to it, unless they want to use the new features. In particular, current DCAT 2 deployments that do not overlap with the DCAT 3 new features (e.g., versioning, dataset series and inverse properties) don't need to change anything to remain in conformance with DCAT 3."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"SCHEMA"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"vocabulary"}),(0,n.jsx)(e.td,{children:"CC BY-SA 3.0"}),(0,n.jsxs)(e.td,{children:["Various formats<br/>",(0,n.jsx)(e.a,{href:"https://schema.org/docs/developers.html",children:"https://schema.org/docs/developers.html"})]}),(0,n.jsx)(e.td,{children:"rdf"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsxs)(e.td,{children:[(0,n.jsx)(e.a,{href:"https://schema.org&lt;br/&gt;Various",children:"https://schema.org&lt;br/&gt;Various"})," formats<br/>",(0,n.jsx)(e.a,{href:"https://schema.org/docs/developers.html",children:"https://schema.org/docs/developers.html"})]}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://schema.org",children:"https://schema.org"})}),(0,n.jsx)(e.td,{children:"schema.org: These vocabularies cover entities, relationships between entities and actions, and can easily be extended through a well-documented extension model."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"PROV-O"}),(0,n.jsx)(e.td,{children:"The PROV Ontology"}),(0,n.jsx)(e.td,{children:"ontology"}),(0,n.jsx)(e.td,{children:"W3C License"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"W3C"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsxs)(e.td,{children:["Namespace: ",(0,n.jsx)(e.a,{href:"http://www.w3.org/ns/prov#&lt;br/&gt;&lt;br/&gt;OWL",children:"http://www.w3.org/ns/prov#&lt;br/&gt;&lt;br/&gt;OWL"})," encoding: ",(0,n.jsx)(e.a,{href:"http://www.w3.org/TR/prov-o/",children:"http://www.w3.org/TR/prov-o/"})]}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.w3.org/TR/prov-o/",children:"https://www.w3.org/TR/prov-o/"})}),(0,n.jsx)(e.td,{children:"w3c: It provides a set of classes, properties, and restrictions that can be used to represent and interchange provenance information generated in different systems and under different contexts. It can also be specialized to create new classes and properties to model provenance information for different applications and domains."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"DCTERMS"}),(0,n.jsx)(e.td,{children:"Dublin Core"}),(0,n.jsx)(e.td,{children:"metadata standard"}),(0,n.jsx)(e.td,{children:"CC BY 4.0"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"rdf"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsxs)(e.td,{children:[(0,n.jsx)(e.a,{href:"http://purl.org/dc/terms/",children:"http://purl.org/dc/terms/"}),"; ",(0,n.jsx)(e.a,{href:"http://purl.org/dc/elements/1.1/",children:"http://purl.org/dc/elements/1.1/"})]}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.dublincore.org/specifications/dublin-core/",children:"https://www.dublincore.org/specifications/dublin-core/"})}),(0,n.jsx)(e.td,{children:"dublincore.org: The Dublin Core\u2122 metadata standard is a simple yet effective element set for describing a wide range of networked resources. The Dublin Core\u2122 standard includes two levels: Simple and Qualified. Simple Dublin Core\u2122 comprises fifteen elements; Qualified Dublin Core\u2122 includes three additional elements (Audience, Provenance and RightsHolder), as well as a group of element refinements (also called qualifiers) that refine the semantics of the elements in ways that may be useful in resource discovery."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"openDS"}),(0,n.jsx)(e.td,{children:"Open Digital Specimens"}),(0,n.jsx)(e.td,{children:"data model"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/DiSSCo/opends",children:"https://github.com/DiSSCo/opends"})}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"GitHub: openDS is a speci\ufb01cation of Digital Specimen and other related object type definitions essential to mass digitization of natural science collections and their digital use in a new generation of infrastructure and applications. For the principal digital object types corresponding to major categories of collections and specimens\u2019 data, openDS de\ufb01nes the structure and content of each object type, and the operations that can act upon them."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"ODS"}),(0,n.jsx)(e.td,{children:"The Ontology for open Digital Specimens (ODS)"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"OBO(?)"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://github.com/DiSSCo/openDS/blob/master/ods-ontology/ods-ont-intro.md",children:"https://github.com/DiSSCo/openDS/blob/master/ods-ontology/ods-ont-intro.md"})}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"GitHub: The Ontology for open Digital Specimens (ODS) situates open Digital Specimens in the relevant OBO Foundry ontologies and extends from those roots to define the new concepts needed to support mass digitization and Digital Specimens on the Internet."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"ABCD-EFG"}),(0,n.jsx)(e.td,{children:"Access to Biological Collection Data Extended for Geosciences."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"CC BY 4.0"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"xsd"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsxs)(e.td,{children:[(0,n.jsx)(e.a,{href:"https://terms.tdwg.org/wiki/ABCD_2&lt;br/&gt;Currently",children:"https://terms.tdwg.org/wiki/ABCD_2&lt;br/&gt;Currently"})," offline<br/><br/>Schema abcdefg.xsd: ",(0,n.jsx)(e.a,{href:"https://efg.geocase.eu/sites/default/documentation/html/efg.html",children:"https://efg.geocase.eu/sites/default/documentation/html/efg.html"})]}),(0,n.jsxs)(e.td,{children:["ABCD Schema is a comprehensive XML-based standard for the access to and exchange of natural history collection and observational data.<br/>ABCD allows a detailed, atomized mapping of information about collection specimens and results of observations in the field. Data from any arbitrarily structured database can be mapped using freely available open source software (BioCASE provider software - ",(0,n.jsx)(e.a,{href:"http://www.biocase.org",children:"www.biocase.org"}),")."]}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"SKOS"}),(0,n.jsx)(e.td,{children:"Simple Knowledge Organisation System"}),(0,n.jsx)(e.td,{children:"terminologies"}),(0,n.jsx)(e.td,{children:"W3C License"}),(0,n.jsx)(e.td,{children:"SKOS"}),(0,n.jsx)(e.td,{children:"skos"}),(0,n.jsx)(e.td,{children:"only indexed"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"http://www.w3.org/2004/02/skos/core#",children:"http://www.w3.org/2004/02/skos/core#"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.w3.org/2004/02/skos/&lt;br/&gt;&lt;br/&gt;https://www.w3.org/TR/skos-reference/",children:"https://www.w3.org/2004/02/skos/&lt;br/&gt;&lt;br/&gt;https://www.w3.org/TR/skos-reference/"})}),(0,n.jsx)(e.td,{children:"SKOS is an area of work developing specifications and standards to support the use of knowledge organization systems (KOS) such as thesauri, classification schemes, subject heading systems and taxonomies within the framework of the Semantic Web."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"WLMO"}),(0,n.jsx)(e.td,{children:"The WildLife Monitoring Ontology"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"developed"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://wildlive.senckenberg.de/wlmo/ontology.ttl",children:"https://wildlive.senckenberg.de/wlmo/ontology.ttl"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://wildlive.senckenberg.de/wlmo/",children:"https://wildlive.senckenberg.de/wlmo/"})}),(0,n.jsx)(e.td,{children:"The ontology developed for representing wildlife monitoring capture events, the setup within which they were observed and their results."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"SSN"}),(0,n.jsx)(e.td,{children:"Semantic Sensor Network Ontology"}),(0,n.jsx)(e.td,{children:"sensors and their observations"}),(0,n.jsx)(e.td,{children:"W3C License"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"W3C_based, DUL aligned, OBOE_aligned, PROV-O_aligned, O&M_aligned"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://bioportal.bioontology.org/ontologies/SSN",children:"https://bioportal.bioontology.org/ontologies/SSN"})}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"http://purl.oclc.org/NET/ssnx/ssn",children:"http://purl.oclc.org/NET/ssnx/ssn"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://www.w3.org/2005/Incubator/ssn/ssnx/ssn&lt;br/&gt;&lt;br/&gt;https://service.tib.eu/ts4tib/ontologies/ssn",children:"https://www.w3.org/2005/Incubator/ssn/ssnx/ssn&lt;br/&gt;&lt;br/&gt;https://service.tib.eu/ts4tib/ontologies/ssn"})}),(0,n.jsx)(e.td,{children:"Bioportal: This ontology describes sensors, actuators and observations, and related concepts. It does not describe domain concepts, time, locations, etc. these are intended to be included from other ontologies via OWL imports."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"SOSA"}),(0,n.jsx)(e.td,{children:"Sensor, Observation, Sample and Actuator (SOSA) Ontology"}),(0,n.jsx)(e.td,{children:"vocabulary"}),(0,n.jsx)(e.td,{children:"W3C License"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"W3C_based, DUL aligned, OBOE_aligned, PROV-O_aligned, O&M_aligned"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"http://liveschema.eu/dataset/lov_sosa",children:"http://liveschema.eu/dataset/lov_sosa"})}),(0,n.jsx)(e.td,{}),(0,n.jsxs)(e.td,{children:["Ghislain Atemezing: This ontology is based on the SSN Ontology by the W3C Semantic Sensor Networks Incubator Group (SSN-XG), together with considerations from the W3C/OGC Spatial Data on the Web Working Group.<br/><br/>Article ",(0,n.jsx)(e.a,{href:"https://doi.org/10.1016/j.websem.2018.06.003",children:"https://doi.org/10.1016/j.websem.2018.06.003"})," : SOSA is the result of rethinking the W3C-XG Semantic Sensor Network (SSN) ontology based on changes in scope and target audience, technical developments, and lessons learned over the past years. SOSA also acts as a replacement of SSN\u2019s Stimulus Sensor Observation (SSO) core.<br/><br/>Observation, sampling and sample, time and more: ",(0,n.jsx)(e.a,{href:"https://www.youtube.com/watch?v=BHBU5gYAmV4",children:"https://www.youtube.com/watch?v=BHBU5gYAmV4"})]}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"AO"}),(0,n.jsx)(e.td,{children:"Annotation Ontology"}),(0,n.jsx)(e.td,{children:"vocabulary"}),(0,n.jsx)(e.td,{children:"CC BY 3.0"}),(0,n.jsx)(e.td,{children:"OWL"}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://code.google.com/archive/p/annotation-ontology/source/default/source",children:"https://code.google.com/archive/p/annotation-ontology/source/default/source"})}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"https://code.google.com/archive/p/annotation-ontology/wikis/Homepage.wiki",children:"https://code.google.com/archive/p/annotation-ontology/wikis/Homepage.wiki"})}),(0,n.jsx)(e.td,{children:"Google Code: Annotation Ontology (AO) is a vocabulary designed to extensively reuse existing domain ontologies (entities annotations or semantic tags) and to provide several other kind of annotations - comments, textual annotation (classic tags), notes, examples, erratum... - on potentially any kind of document (text, images, audio...) and document fragments."}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"GEOSPARQL"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:'OGC "open" license'}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"imports Person and Organization from SCHEMA"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"http://www.opengis.net/ont/geosparql",children:"http://www.opengis.net/ont/geosparql"})}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"FOAF"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"W3C License"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"http://xmlns.com/foaf/0.1",children:"http://xmlns.com/foaf/0.1"})}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"VCARD"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"W3C License"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"http://www.w3.org/2006/vcard/ns",children:"http://www.w3.org/2006/vcard/ns"})}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]}),(0,n.jsxs)(e.tr,{children:[(0,n.jsx)(e.td,{children:"T4FS"}),(0,n.jsx)(e.td,{children:"terms4FAIRskills"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"CC BY 4.0"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:"owl"}),(0,n.jsx)(e.td,{children:"reused"}),(0,n.jsx)(e.td,{children:"imports DC, OWL, DCTERMS, OMO, RDFS, IAO, OIO, BFO, RO"}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{children:(0,n.jsx)(e.a,{href:"http://purl.obolibrary.org/obo/t4fs.owl",children:"http://purl.obolibrary.org/obo/t4fs.owl"})}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{}),(0,n.jsx)(e.td,{})]})]})]})]})}function c(t={}){const{wrapper:e}={...(0,i.R)(),...t.components};return e?(0,n.jsx)(e,{...t,children:(0,n.jsx)(h,{...t})}):h(t)}},8453:(t,e,o)=>{o.d(e,{R:()=>d,x:()=>r});var s=o(6540);const n={},i=s.createContext(n);function d(t){const e=s.useContext(i);return s.useMemo(function(){return"function"==typeof t?t(e):{...e,...t}},[e,t])}function r(t){let e;return e=t.disableParentContext?"function"==typeof t.components?t.components(n):t.components||n:d(t.components),s.createElement(i.Provider,{value:e},t.children)}}}]);